formfyxer/data/simplified_words.yml

```diff
@@ -1,199 +1,459 @@
+in the neighbourhood of: "[about, around]"
 under the provisions of: under
+acquaint yourself with: "[find out about, read]"
 interpose no objection: don't object
 afford an opportunity: "[allow, let]"
+despite the fact that: "[though, although]"
+give consideration to: "[consider, think about]"
 has a requirement for: needs
+have the authority to: may
 is in consonance with: "[agrees with, follows]"
+on numerous occasions: often
 provides guidance for: guides
 successfully complete: "[complete, pass]"
 this activity command: "[us, we]"
 with the exception of: except for
-due to the fact that: "[due to, since, because]"
+despatch or dispatch: "[send, post]"
+due to the fact that: "[due to, since, because, as]"
 effect modifications: "[make changes]"
+in a number of cases: "[some, say how many]"
+in the event of/that: if
 in view of the above: so
+on the occasion that: "[when, if]"
+please find enclosed: I enclose
 adversely impact on: "[hurt, set back]"
-at the present time: now
+as a consequence of: because
+at the present time: "[now, (omit)]"
+for the duration of: "[during, while]"
+for the reason that: because
+in conjunction with: "[and, with]"
+on the grounds that: because
 pertaining to about: "[of, on]"
+that being the case: if so
+we have pleasure in: we are glad to
+along the lines of: "[like, as in]"
 combat environment: combat
+for the purpose of: "[to, for]"
 in a timely manner: "[on time, promptly]"
-in accordance with: by
-in the near future: "[shortly, soon]"
+in accordance with: "[as under, in line with, because of, by]"
+in connection with: "[for, about]"
+in the near future: soon
 is responsible for: "[(omit), handles]"
 on a regular basis: (omit)
+subsequent to/upon: after
+to the extent that: "[if, when]"
 until such time as: until
+as of the date of: from
+at its discretion: "[can, may, (omit)]"
 during the period: during
+during which time: while
+in the absence of: without
 in the process of: (omit)
+is of the opinion: thinks
+may in the future: "[may, might, could]"
 with reference to: about
+you are requested: please
 adversely impact: "[hurt, set back]"
 as prescribed by: "[in, under]"
+at an early date: "[soon, say when]"
+costs the sum of: costs
+hold in abeyance: "[wait, postpone]"
 in the amount of: for
+in the course of: "[while, during]"
 is applicable to: applies to
 is authorized to: may
+it is compulsory: you must
+it is known that: I/we know that
 state of the art: latest
 state-of-the-art: latest
+with effect from: from
+attributable to: "[due to, because of]"
 close proximity: near
 for a period of: for
 in an effort to: to
 in the event of: if
+irrespective of: "[despite, even if]"
 it is essential: "[must, need to]"
+it is mandatory: you must
 it is requested: "[please, we request, I request]"
-notwithstanding: "[despite, still]"
+large number of: "[many, most, say how many]"
+notwithstanding: "[even if, despite, still, yet]"
 shall be issued: will be sent # still passive but can't fix in isolation
```

Comment:
Could do something like "the organization will send", like how "and/or" has "a or b or both".